scripts/CraftTweaker/Pyrotech/Pyrotech.zs

```diff
@@ -48,14 +50,10 @@ rh(<pyrotech:crude_hammer>);
 h(<pyrotech:material:27>);
 <ore:stickStone>.remove(<pyrotech:material:27>);
 
-recipes.remove(<pyrotech:sawmill_blade_stone>);
-recipes.addShaped(<pyrotech:sawmill_blade_stone>,
-   [[<ore:rocks>, <ore:rocks> , <ore:rocks>],
-    [<ore:rocks>, <ore:stones>, <ore:rocks>],
-    [<ore:rocks>, <ore:rocks> , <ore:rocks>]]);
+RecipeUtils.tweakRecipe(true, <pyrotech:sawmill_blade_stone>,
+    RecipeUtils.createSurround(<ore:stones>, <ore:rocks>));
 
-recipes.remove(<pyrotech:flint_and_tinder>);
-recipes.addShapeless(<pyrotech:flint_and_tinder>, [<pyrotech:material:10>, <pyrotech:material:13>, <divergentunderground:rock_stone>]);
+RecipeUtils.tweakRecipe(false, <pyrotech:flint_and_tinder>, [[<pyrotech:material:10>, <pyrotech:material:13>, <ore:rock>]]);
```

Comment:
That quartzite rock here was intentional, but it's good. 
I'm planning to make an oredict for rocks suited for making fire, like those with a lot of mineral quartz.